### PR TITLE
add title scene

### PR DESCRIPTION
**Pull request by ajapon88**
T/O

### CODE REVIEW

File: SampleProject/Assets/Scripts/Title.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Title : MonoBehaviour {
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1600] The class must have a documentation header.

File: SampleProject/Assets/Scripts/Title.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+	// Use this for initialization
+	void Start () {
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1600] The method must have a documentation header.

File: SampleProject/Assets/Scripts/Title.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+		
+	}
+	
+	// Update is called once per frame
+	void Update () {
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1600] The method must have a documentation header.

File: SampleProject/Assets/Scripts/Title.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1633] The file has no header, the header Xml is invalid, or the header is not located at the top of the file.

File: SampleProject/Assets/Scripts/Title.cs (hunk 1 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 1 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1505] An opening curly bracket must not be followed by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1505] An opening curly bracket must not be followed by a blank line.

File: SampleProject/Assets/Scripts/Title.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Title : MonoBehaviour {
+
+	// Use this for initialization
+	void Start () {
+		
+	}
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1508] A closing curly bracket must not be preceded by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1505] An opening curly bracket must not be followed by a blank line.

File: SampleProject/Assets/Scripts/Title.cs [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+	
+	// Update is called once per frame
+	void Update () {
+		
+	}
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1508] A closing curly bracket must not be preceded by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1400] The method must have an access modifier.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1400] The method must have an access modifier.

File: SampleProject/Assets/Scripts/Title.cs [hunk 7]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Title : MonoBehaviour {
+
+	// Use this for initialization
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1008] Invalid spacing around the opening parenthesis.

File: SampleProject/Assets/Scripts/Title.cs [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+	void Start () {
+
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 5 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Title : MonoBehaviour {
+
+	// Use this for initialization
+	void Start () {
+		
+	}
+
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+using UnityEngine;
+
+public class Title : MonoBehaviour {
+
+	// Use this for initialization
+	void Start () {
+		
+	}
+	
+	// Update is called once per frame
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1008] Invalid spacing around the opening parenthesis.

File: SampleProject/Assets/Scripts/Title.cs [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+	void Update () {
+
```
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 6 above)
Comment (nishikawat-mixi) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 1 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1600] The class must have a documentation header.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1600] The method must have a documentation header.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1600] The method must have a documentation header.

File: SampleProject/Assets/Scripts/Title.cs (hunk 4 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1633] The file has no header, the header Xml is invalid, or the header is not located at the top of the file.

File: SampleProject/Assets/Scripts/Title.cs (hunk 1 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 1 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1505] An opening curly bracket must not be followed by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1505] An opening curly bracket must not be followed by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 5 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1508] A closing curly bracket must not be preceded by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1505] An opening curly bracket must not be followed by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 6 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1508] A closing curly bracket must not be preceded by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1400] The method must have an access modifier.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1400] The method must have an access modifier.

File: SampleProject/Assets/Scripts/Title.cs (hunk 7 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1008] Invalid spacing around the opening parenthesis.

File: SampleProject/Assets/Scripts/Title.cs (hunk 8 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 5 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 9 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 10 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1008] Invalid spacing around the opening parenthesis.

File: SampleProject/Assets/Scripts/Title.cs (hunk 11 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 6 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 1 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1600] The class must have a documentation header.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1600] The method must have a documentation header.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1600] The method must have a documentation header.

File: SampleProject/Assets/Scripts/Title.cs (hunk 4 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1633] The file has no header, the header Xml is invalid, or the header is not located at the top of the file.

File: SampleProject/Assets/Scripts/Title.cs (hunk 1 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 1 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1505] An opening curly bracket must not be followed by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1505] An opening curly bracket must not be followed by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 5 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1508] A closing curly bracket must not be preceded by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs]: [SA1505] An opening curly bracket must not be followed by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 1 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:5]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:5]: [SA1505] An opening curly bracket must not be followed by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 7 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:7]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 2 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:8]: [SA1400] The method must have an access modifier.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:8]: [SA1600] The method must have a documentation header.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:8]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:8]: [SA1505] An opening curly bracket must not be followed by a blank line.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:8]: [SA1027] Tabs are not allowed. Use spaces instead.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:8]: [SA1008] Invalid spacing around the opening parenthesis.

File: SampleProject/Assets/Scripts/Title.cs (hunk 8 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:9]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 5 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:10]: [SA1027] Tabs are not allowed. Use spaces instead.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:10]: [SA1508] A closing curly bracket must not be preceded by a blank line.

File: SampleProject/Assets/Scripts/Title.cs (hunk 9 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:11]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 10 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:12]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 3 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:13]: [SA1505] An opening curly bracket must not be followed by a blank line.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:13]: [SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:13]: [SA1400] The method must have an access modifier.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:13]: [SA1600] The method must have a documentation header.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:13]: [SA1027] Tabs are not allowed. Use spaces instead.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:13]: [SA1008] Invalid spacing around the opening parenthesis.

File: SampleProject/Assets/Scripts/Title.cs (hunk 11 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:14]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 6 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:15]: [SA1508] A closing curly bracket must not be preceded by a blank line.

Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:15]: [SA1027] Tabs are not allowed. Use spaces instead.

File: SampleProject/Assets/Scripts/Title.cs (hunk 4 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:1]: [SA1633] The file has no header, the header Xml is invalid, or the header is not located at the top of the file.

File: SampleProject/Assets/Scripts/Title.cs (hunk 1 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:5]: [SA1600] The class must have a documentation header.

**Comment by ajapon88** [comment]
houd style keeper

**Comment by ajapon88** [comment]
hound style keeper

**Comment by ajapon88** [comment]
hound style keeper

**Comment by ajapon88** [comment]
hound style keeper

**Comment by ajapon88** [comment]
hound style keeper

**Comment by ajapon88** [comment]
hound style keeper

**Comment by ajapon88** [comment]
hound style keeper

**Comment by ajapon88** [comment]
hound style keeper

**Comment by ajapon88** [comment]
hound style keeper

File: SampleProject/Assets/Scripts/Title.cs (hunk 4 above)
Comment (ajapon88) [inline SampleProject/Assets/Scripts/Title.cs:1]: [SA0102] A syntax error has been discovered in file /var/folders/80/vq2yfwcx0d541ghj3n_6qn_h0000gn/T/style_keeper/8d689f42372a6888604fce36ba4bd6dba8ee5c07/SampleProject/Assets/Scripts/Title.cs on line 1.

**Pull request merged**